index.html

```diff
@@ -288,12 +288,13 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
     <div class="row">
     <div class="row mb64 mb-xs-24">
 		                <div class="col-sm-12 text-center spread-children-large">
-		                    <img class="col-sm-2" alt="Miguel Piedrafita" src="img/m1guelpf.png">
+							<img class="col-sm-2" alt="Miguel Piedrafita" src="img/m1guelpf.png">
```

Comment:
Let's keep it professional, don't deintedn the code, i meant the blank lines.

Reply:
no. the line no blank.

Reply:
See the giff. You are adding some spaces before the <img> tag :)